fire/__init__.py

```diff
@@ -12,12 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Python Fire module for third_party."""
+"""The Python Fire module."""
 
 from __future__ import absolute_import
 from __future__ import division
 from __future__ import print_function
 
 from fire.core import Fire
+from fire.core import FireExit
 
-__all__ = ['Fire']
+__all__ = ['Fire', 'FireExit']
```

Comment:
I think this is a fair point and agree with it (tho maybe nice to be able to say "exit status of 1 means client exception, exit status of 2 means bad parameter or fire invocation") - I went ahead in a PR and added a helper to check output from FireExit anyways.